### PR TITLE
Fix lint/build errors.

### PR DESCRIPTION
**Pull request by klassenjs**
This should get applied before #558 and #559 then those will likely need to be rebased.

### CODE REVIEW

File: src/gm3/components/map.js [hunk 1]
```diff
@@ -498,7 +496,8 @@ class Map extends React.Component {
             // setup the spatial filter.
             query_params.geometryType = geom_type_lookup[queryGeometry.type];
             query_params.geometry = esri_format.writeGeometry(ol_geom);
-            query_params.spatialRel = 'esriSpatialRelIntersects'; // for lines?:'esriSpatialRelEnvelopeIntersects';
+            query_params.spatialRel = 'esriSpatialRelIntersects';
+            // for lines?:'esriSpatialRelEnvelopeIntersects';
```
Comment (theduckylittle) [inline src/gm3/components/map.js:500]: This should likely go above the query_params line.

Comment (klassenjs) [reply src/gm3/components/map.js:500]: Really, it should be implemented or deleted.

Comment (klassenjs) [reply src/gm3/components/map.js:500]: It's effectively a TODO: for an unimplemented case.

Comment (klassenjs) [reply src/gm3/components/map.js:500]: It doesn't describe what the line above it does, it describes future work.  That's why I put it below.

**Comment by theduckylittle** [comment]
Cool. :+1:

**Pull request merged**